plugins/direct_events_plugin/db/services.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+defined('MOODLE_INTERNAL') || die();
+
+$services = array(
+      'directeventhandler' => array(                                                // the name of the web service
+          'functions' => array ('local_direct_event_handler'), // web service functions of this service
+          'requiredcapability' => '',                // if set, the web service user need this capability to access 
+                                                                              // any function of this service. For example: 'some/capability:specified'                 
+          'restrictedusers' => 0,                                             // if enabled, the Moodle administrator must link some user to this service
+                                                                              // into the administration
+          'enabled' => 1,                                                       // if enabled, the service can be reachable on a default installation
+          'shortname' =>  'd_handler',       // optional – but needed if restrictedusers is set so as to allow logins.
+          'downloadfiles' => 0,    // allow file downloads.
+          'uploadfiles'  => 0,      // allow file uploads.
+          'ajax' => true
```

Comment:
As part of the spike work, did we find a way to confirm we can access / exercise the functions in this web service from Moodle FE code? (e.g. Javascript run in the browser)

Reply:
Resolving per [this comment](https://github.com/openstax/k12/issues/183#issuecomment-1167818554)